code/modules/shuttle/ert.dm

```diff
@@ -6,14 +6,13 @@
 	resistance_flags = INDESTRUCTIBLE
 	flags = NODECONSTRUCT
 
-/obj/machinery/computer/shuttle/ert/Topic(href, href_list)
-	if(href_list["move"])
+/obj/machinery/computer/shuttle/ert/tgui_act(action, list/params, datum/tgui/ui, datum/tgui_state/state)
+	if(action == "move")
 		var/authorized_roles = list(SPECIAL_ROLE_ERT, SPECIAL_ROLE_DEATHSQUAD)
 		if(!((usr.mind?.assigned_role in authorized_roles) || is_admin(usr)))
 			message_admins("Potential ERT shuttle hijack, ERT shuttle moved by unauthorized user: [key_name_admin(usr)]")
 	..()
```

Comment:
had a discussion with AA on this on discord, regarding not having if(..()) return at the start. He said "
 there isnt any better way to do this
 shuttles suck, whats new" So leaving it like this for the time being.